slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/dao/ValidatorsDaoTest.java

```diff
@@ -88,6 +88,20 @@ public void storesUnregisteredValidatorsInDb(final Handle handle) {
     assertThat(validators.get(4)).isEqualToComparingFieldByField(new Validator(5, Bytes.of(104)));
   }
 
+  @Test
+  public void registeringValidatorsDoesNotEnableDisabledValidators(final Handle handle) {
```

Comment:
Can you point me to the code that prevents disabled validators from being reenabled?

Reply:
There is no specific code for this, it is achieved by not making any changes to the validator registration. In retrospect I think this code should go. The reason it's not enabling validators is because we don't registered validator that already exist. So I don't think this test case makes sense anymore. Originally I was taking a different approach.
Removing the test.